aiida/cmdline/commands/cmd_code.py

```diff
@@ -235,6 +236,27 @@ def show(code):
     echo.echo(tabulate.tabulate(table))
 
 
+@verdi_code.command()
+@arguments.CODE()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@with_dbenv()
+def export(code, output_file):
+    """Export code to a yaml file."""
+    cli_options = code._get_cli_options()
+    code_data = {}
+
+    for key in cli_options.keys():
+        if key == 'computer':
+            value = getattr(code, key).label
+        else:
+            value = getattr(code, key)
+
+        code_data[key] = str(value)
```

Comment:
Do we need to convert to `str`? Should the yaml serializer not take care of this. Then the boolean question would be solved in any case. I don't think we have codes that have any attributes that are not YAML-serializable. I think the data is guaranteed to be JSON-serializable, since it comes from the database, so it should also be YAML-serializable.

Reply:
The problem comes from the `filepath_executable` which I read by `code._get_cli_options`. The type is `PurePosixPath` which is not yaml-serializable. I can make another elif for this, but I think should be fine that all converted to string?

Reply:
Yeah I agree, fine to keep the cast to str then as long as the round-trip works